### DISCUSSION TITLE
How to build the flet code and create single executable file?

### DISCUSSION THREAD

**Discussion by vinaysharmaaddy143** [flet-dev/flet#362] [Q&A] [answered]
Hello,

Is there any way to create a single executable file for Flet code. For example, create an .exe file for windows desktop app created using flet.

Actually I am new in this and didn't see any information on this topic. Can you please help me in this?

**Comment by FeodorFitsner** [comment, marked as answer]
This is the guide: https://flet.dev/docs/guides/python/packaging-desktop-app

**Reply by vinaysharmaaddy143** [reply]
Thanks for your reply. One more query i have.

How to remove/replace the flet logo from my app?

**Reply by ndonkoHenri** [reply]
When packaging your executable file: 
[https://flet.dev/docs/guides/python/packaging-desktop-app/#customizing-package-icon](https://flet.dev/docs/guides/python/packaging-desktop-app/#customizing-package-icon)

Adding a custom Splash screen logo:
[https://github.com/flet-dev/flet/discussions/99#discussioncomment-3218348](https://github.com/flet-dev/flet/discussions/99#discussioncomment-3218348)

**Reply by vinaysharmaaddy143** [reply]
> When packaging your executable file: https://flet.dev/docs/guides/python/packaging-desktop-app/#customizing-package-icon
> 
> Adding a custom Splash screen logo: [#99 (comment)](https://github.com/flet-dev/flet/discussions/99#discussioncomment-3218348)

Thanks
